test/index.test.js

```diff
@@ -780,8 +780,19 @@ describe('Segment.io', function() {
         segment.options.crossDomainIdServers = [
           'userdata.example1.com',
           'xid.domain2.com',
-          'localhost'
```

Comment:
We had `localhost` in here since we weren't faking `window.location.hostname` which returns localhost. Now that we are faking it, we don't need it.